### PR TITLE
Rtkvs put fixes

### PR DESCRIPTION
**Pull request by SergeyKaydalov**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.
- [ ] Change modifies documentation or comments only.

#### Description
1. Improved put/get latency esitmation
2. A lot of fixes for RTKVS reptrans driver and its backends. 

#### Which issue this PR fixes

fixes #

### CODE REVIEW

**Comment by dyusupov** [comment]
:star: PR built and available in a preview environment **nexenta-edgefs-pr-370**

File: src/ccow/src/libreptrans/unnamed-put.c [hunk 1]
```diff
@@ -1064,23 +1064,34 @@ unnamedput_srv__put_proposal(struct state *st)
...
+
+		if (timeout > PP_WEIGHT_PAUSE_MS)
+			timeout = PP_WEIGHT_PAUSE_MS;
+
 		req->pp_timer_req->data = req;
-		uv_timer_start(req->pp_timer_req,
-		    unnamedput_srv__pp_busy_timeout, PP_WEIGHT_PAUSE_MS, 0);
-		return;
+		uv_timer_start(req->pp_timer_req, unnamedput_srv__pp_busy_timeout,
+			timeout, 0);
```
Comment (dyusupov) [inline src/ccow/src/libreptrans/unnamed-put.c]: we missing return here - typo?

Comment (SergeyKaydalov) [reply src/ccow/src/libreptrans/unnamed-put.c]: Did a final refactoring and missed.

**Comment by dyusupov** [comment]
:star: PR built and available in a preview environment **nexenta-edgefs-pr-370**

**Pull request merged**